src/chargen.cpp

```diff
@@ -957,7 +988,7 @@ int parse_assign(struct descriptor_data *d, const char *arg)
     case PR_MAGIC:
       break;
     case PR_RESOURCE:
-      GET_NUYEN_RAW(d->character) = nuyen_vals[d->ccr.temp];
+      GET_NUYEN_RAW(d->character) = MIN(nuyen_vals[d->ccr.temp], d->ccr.is_otaku ? 5000 : nuyen_vals[0]);

```

Comment:
What's the reason for / effect of this line? It seems to cap your nuyen to 5k if you're an otaku no matter what. If Otaku can never start with more than 5k, please disable setting the nuyen resources priority for otaku on any but the lowest slot to avoid creating a newbie trap.

Reply:
Correct, otaku can never start with more than 5k per the book. But it has to be settable as higher otherwise there aren't enough options to spend all priority points / etc.

Reply:
For the record I also made this not a newbie trap in that no matter what you select it will show 5k in the interface.

Reply:
![image](https://github.com/user-attachments/assets/5cdae3d9-af93-4a37-8229-fea809cc1e62)